### DISCUSSION TITLE
How to do resize first and then upload ?

### DISCUSSION THREAD

**Discussion by vishalinfyom** [livewire/livewire#5284] [1. Help] [answered]
we want to resize first and then upload it to direct s3.

Right now we are using a direct uploaded file object.

![image](https://user-images.githubusercontent.com/33887755/193525970-b84f73c1-c55b-450b-ae5b-3ca176dca76c.png)

**Comment by ousid** [comment]
@vishalinfyom, what's wrong exactly here? I see you are storing the file twice! Have you tried to override the logo property with the new resized object?

Not sure, but maybe:
```php
if (...) {
 $this->logo = Image::make(...);
// without save(...);
}

// then 
$this->model->logo_invoice ....
```

**Comment by vishalinfyom** [comment, marked as answer]
That is sorted now, i just created object of `UploadedFile` and its works
